### PR TITLE
Correctly generate metadata for zoom levels

### PR DESCRIPTION
**Pull request by dispiste**

### CODE REVIEW

File: landez/tiles.py [hunk 1]
```diff
@@ -259,7 +259,11 @@ def zoomlevels(self):
         """
         Return the list of covered zoom levels
         """
-        return self._bboxes[0][1]  #TODO: merge all coverages
+        zooms = set()
+        for coverage in self._bboxes:
+            for zoom in coverage[1]:
+                zooms.add(zoom)
+        return list(zooms)
```
Comment (leplatrem) [inline landez/tiles.py]: Shouldn't we sort them ?

**Comment by leplatrem** [comment]
Thanks for contributing !

It looks good to me except that I would have expected a (even small) unit test...

**Comment by dispiste** [comment]
OK, I think it is ready now.

**Comment by leplatrem** [comment]
Thanks for updating the test !

It looks good to me! Please wait for makina staff to merge and release this :)

**Pull request merged**